resources/templates/slack.mustache.json

```diff
@@ -0,0 +1,7 @@
+{
+    "username": "runbld",
```

Comment:
@drewr I just realized I still have the default slack username set to `runbld`.  This can be anything we want. I don't think `runbld` makes sense. 

What do you suggest? How about "ci"?